packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -129,16 +129,20 @@ abstract class BindingBase {
     }
 
     assert(() {
+      const String platformOverride = 'platformOverride';
```

Comment:
nit: call this platformOverrideExtensionName or something a little longer instead of platformOverride.

Reply:
Done.